hack/local-development/remote-garden/enable-seed-authorizer

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source $(dirname "${0}")/../common/helpers
+
+SEED_KUBECONFIG=$1
```

Comment:
Does it make sense to use the standard `KUBECONFIG` variable and eliminate this argument?

Reply:
We need both kubeconfigs, the one for the remote garden and the one for its seed. The one for the remote garden is used to get the pod IP of the `quic-server` pod and create / delete cluster roles and rolebindings. For consistency with other scripts, e.g. `open-gardener-tunnels`, I would prefer the `KUBECONFIG` to point to the remote garden, and so the seed kubeconfig should be passed as an argument.

Reply:
I see, makes sense, thanks!